test/__snapshots__/TypeScriptGenerator-test.ts.snap

```diff
@@ -568,6 +582,7 @@ export type PlainUserNameRenderer_name$key = {
 
 
 // MarkdownUserNameRenderer_name.graphql
+import { FragmentRefs } from "relay-runtime";
```

Comment:
@ds300 Were we not already emitting these imports for e.g. the below `FragmentRefs<"MarkdownUserNameRenderer_name">` part? I see in our code-base that these imports exist, but unsure why then these tests did not have them yet 🤔